source/_integrations/androidtv.markdown

```diff
@@ -179,14 +66,26 @@ When connecting to your device for the first time, a dialog will appear on your
 
 ### 1. Python ADB Implementation
 
-The default approach is to connect to your device using the `adb-shell` Python package. As of Home Assistant 0.101, if a key is needed for authentication and it is not provided by the `adbkey` configuration option, then Home Assistant will generate a key for you.
+The default approach is to connect to your device using the `adb-shell` Python package. As of Home Assistant 0.101, if a key is needed for authentication and it is not provided by the `ADB Key` setup option, then Home Assistant will generate a key for you.
```

Comment:
Is providing an ADB key a hidden/advanced configuration option? After all, it's easier and simpler to just let HA generate a key.

Reply:
Yes is advanced option, and is specified in the next chapter `Python ADB Implementation`